corehq/apps/hqadmin/views.py

```diff
@@ -838,3 +838,20 @@ def view_data(case_id, indicators):
         "doc_id": doc_id
     }
     return render(request, "hqadmin/callcenter_test.html", context)
+
+
+@require_superuser
+def malt_as_csv(request):
+    from django.core.exceptions import ValidationError
+    from corehq.apps.data_analytics.views import malt_csv_response
+
+    if 'year_month' in request.GET:
+        try:
+            year, month = request.GET['year_month'].split('-')
+            year, month = int(year), int(month)
+            return malt_csv_response(month, year)
+        except ValueError, VaidationError:
```

Comment:
this is legit. you need parens around the the exception list. The code you have is equivalent to
```python
except ValueError as VaidationError:
```